### PR TITLE
feature/todo-model++ 🤙 

### PR DESCRIPTION
**Pull request by markhilb**
* Removed unnessecary compilerOption
🤪

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -7,4 +11,10 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'Neptun';
+  todoService: TodoService = new TodoService();
+
```
Comment (jonjohansen) [inline src/app/app.component.ts:17]: Newlines 🔥

File: src/app/models/todo-model.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
+
```
Comment (jonjohansen) [inline src/app/models/todo-model.ts]: Newlines 🔥

File: src/app/todo.service.ts [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+import todoSamples from '../assets/todo-samples.json';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TodoService {
+
+  todos: TodoModel[] = todoSamples;
+
+  constructor() {
```
Comment (jonjohansen) [inline src/app/todo.service.ts]: If constructor is unused you can wrap it like:
```constructor(){ }```

File: src/app/todo.service.ts [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+  constructor() { 
+
+  }
+
+  addTodo(todo: TodoModel) {
+    this.todos.push(todo);
+  }
+
+  testAddTodo() {
```
Comment (jonjohansen) [inline src/app/todo.service.ts:19]: Are these mock or test values? 🏭

File: src/app/todo.service.ts [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+          x.deadline = edit;
+        }
+        else if(variable === "priority")
+        {
+          x.priority = edit;
+        }
+      }
+    });
+  }
+
```
Comment (jonjohansen) [inline src/app/todo.service.ts]: View logic such as sorting should be handled by the respective component and not necessarily the service.

File: tsconfig.json [hunk 6]
```diff
@@ -1,6 +1,8 @@
 {
   "compileOnSave": false,
   "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
```
Comment (jonjohansen) [inline tsconfig.json]: What does this do?

**Review by jonjohansen — CHANGES_REQUESTED** [review]
Tight, but needs some fixes / explanations. 😄

**Review by Joaalslie — DISMISSED** [review]

**Review by agfagerbakk — DISMISSED** [review]

File: tsconfig.json (hunk 6 above)
Comment (jonjohansen) [reply tsconfig.json]: Fix this.

**Review by jonjohansen — CHANGES_REQUESTED** [review]
Still a few points

**Review by jonjohansen — APPROVED** [review]
👍

**Pull request merged**